tools/debugger/debugger_tester.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+ESCARGOT=$1
+TEST_CASE=$2
+COMMANDS=$3
+EXPECTED=$4
+DEBUGGER_CLIENT=$5
+START_DEBUG_SERVER="${ESCARGOT} --start-debug-server ${TEST_CASE} &"
+three=3
+
+echo "$START_DEBUG_SERVER"
+eval "$START_DEBUG_SERVER"
+sleep 2s
+
+RESULT_TEMP=`mktemp ${TEST_CASE}.out.XXXXXXXXXX`
+
+(cat "${COMMANDS}" | ${DEBUGGER_CLIENT}  --display  ${three}) >${RESULT_TEMP} 2>&1
```

Comment:
I am not sure `--display 3` needs to be enabled by default. If needed, it can be enabled in the test.